narratives/ncov-19B_resurgence.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Resurgence of SARS-CoV-2 19B lineage
+authors:
+  - Cassia Wagner
+  - Emma Hodcroft
+  - Richard Neher
+  - Trevor Bedford
+
+authorLinks:
+  - https://bedford.io/team/cassia-wagner/
+  - https://www.ispm.unibe.ch/about_us/staff/hodcroft_emma/index_eng.html#pane1013282
+  - https://neherlab.org/richard-neher.html
+  - https://bedford.io/team/trevor-bedford/
+affiliations: "Fred Hutch, Seattle, USA; ISPM, Bern, Switzerland, Biozentrum, Basel, Switzerland"
+
+license: "CC-BY"  
+licenseLink: "https://creativecommons.org/licenses/by/4.0/"
+dataset: "https://nextstrain.org/groups/blab/ncov/19B?d=tree,map,entropy&legend=open"
+date: "2021 February 11"
+
+abstract: "Clade 19B viruses which had largely disappeared from circulation have slightly increased in frequency over the past two months.
+Here we explore the genetic composition of viruses currently circulating in 19B. We find numerous examples of convergent evolution in the spike protein, which may contribute to its increasing frequency."
+---
+
+<!-- Comment tags like these are not rendered, they're just helpful for you -->
+<!-- Known 'gotcha' bug: ensure that links always end in a 'letter' (a period counts). If some kind of text doesn't follow them, it breaks the slide. -->
+
+<!-- SLIDE 1 -->
+<!--  Each slide MUST start with a link to a specific view of the dataset (must match the `dataset` specified above) -->
+# [Resurgence of clade 19B](https://nextstrain.org/ncov/global?d=tree&p=full)
+
+<!-- This is left-side text -->
+Here we see the SARS-CoV-2 global phylogeny colored by viral clade.
+Clades 19A & 19B, in purple and dark blue at the bottom of the tree, predominated early in the pandemic.
+However, these clades were soon eclipsed by viruses in clade 20.
+
+Clade 20 probably dominated for multiple reasons:
+This clade was exported globally, especially to Europe, as early as January 2020 where it spread rapidly undetected for over a month prior to lockdowns.
+Additionally, spike D614G is one of the clade-defining mutations, and this substitution likely confers [increased transmissibility](https://pubmed.ncbi.nlm.nih.gov/32697968/).
+
+Regardless, by June 2020, less than 10% of circulating virus were from clades 19A & 19B. By mid-September, frequency was no more than 1%.
+
+However, transmission did continue at low levels. Recently, we have observed a slight increase in the frequency of 19B.
+In December and January, we estimated 3% of all viruses to be 19B.
+
+This resurgence of a viral clade that had largely died out, especially during the worst phase of the pandemic, is surprising.
```

Comment:
Not entirely sure the meaning of 'during the worst phase of the pandemic' here. If it is the worst phase, naively one might expect that's _exactly_ when one would expect to see older lineages resurging. On the other hand, WHO estimates globally case numbers have gone _down_ in the past few weeks!

Perhaps something more specific (if this is what you meant?) like: "... that had largely died out, particularly when multiple new variants of concern from the 20 Clades are displacing other variants in many countries, is surprising."

Reply:
Yes, the latter is exactly what I meant. Thank you for the helpful rephrasing.